### DISCUSSION TITLE
Running Pocketbase and Flask app at the same time.

### DISCUSSION THREAD

**Discussion by 0xRollerboy** [pocketbase/pocketbase#3213] [Q&A] [answered]
Hi guys. 👋

I'm new to coding, and really don't know how things work yet. 

I've obtanied a certificate from certbot. VPS on Vultr. Cloudflare enabled on full-strict. 

What I struggle with is to start Pocketbase at the start and run my flask app with Gunicorn.

When I manually run ./pocketbase serve - it works fine. But

Thank for everyone:)

Service file:
[Service]
Type           = simple
User           = root
Group          = root
LimitNOFILE    = 4096
Restart        = always
RestartSec     = 5s
StandardOutput = append:/root/pb/errors.log
StandardError  = append:/root/pb/errors.log
ExecStart      = /root/pb/pocketbase serve --http="127.0.0.1:8090/pocketbase/" --https="127.0.0.1:8090/pocketbase/"

NGINX default:
server {
    listen 80;
    server_name fbookmarks.xyz;

    location / {
        proxy_pass http://127.0.0.1:5051;
        include proxy_params;
    }

    location /pocketbase/ {
        proxy_set_header Connection '';
        proxy_http_version 1.1;
        proxy_read_timeout 600s;

        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;

        proxy_pass http://127.0.0.1:8090/;
    }

    # SSL Configuration
    listen 443 ssl;
    ssl_certificate /etc/letsencrypt/live/fbookmarks.xyz/fullchain.pem;
    ssl_certificate_key /etc/letsencrypt/live/fbookmarks.xyz/privkey.pem;
    include /etc/nginx/snippets/ssl-params.conf;  # Optional, but recommended
}

NGINX pockatbase config
server {
    if ($host = fbookmarks.xyz) {
        return 301 https://$host$request_uri;
    } # managed by Certbot

    listen 80;
    server_name fbookmarks.xyz;
    return 404; # managed by Certbot

}

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not entire sure that I understand the above configs, but if you want PocketBase to be accessible under a subpath, you'll need a rewrite rule. You can add above the `proxy_pass ...` line something like:
```
rewrite /pocketbase/(.*) /$1  break;
```

**Reply by 0xRollerboy** [reply]
The basic idea is that I'd run my Flask app.py under domain.com/ 
and pocketbase under domain.com/pocketbase/_/ for admin.

But the exact path doesnt really matter.
